tsh on MacOS crashes with segmentation fault

Trying out teleport - the teleport nodes are all elsewhere, but using tsh on my MacBook.  I am using 'Teleport v1.1.0 git:v1.1.0-0-ga2c7b3c' (the released version).  Connection works fine, I am working for awhile, then : 

dial tcp <nodeip>:3025: getsockopt: connection timed out
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xd0 pc=0x40aeb6b]

  goroutine 40 [running]:
                         panic(0x443d500, 0xc420014090)
                                                       	/usr/local/go/src/runtime/panic.go:500 +0x1a1
                                                                                                     github.com/gravitational/teleport/lib/client.(*NodeSession).updateTerminalSize(0xc42022a0e0, 0xc4201ee240)
                                                                                         	/Users/ekontsevoy/go/src/github.com/gravitational/teleport/lib/client/session.go:281 +0x57b
                                                                     created by github.com/gravitational/teleport/lib/client.(*NodeSession).allocateTerminal
                                      	/Users/ekontsevoy/go/src/github.com/gravitational/teleport/lib/client/session.go:224 +0x7b0

--- a/lib/client/session.go
+++ b/lib/client/session.go
@@ -245,6 +245,7 @@ func (ns *NodeSession) updateTerminalSize(s *ssh.Session) {
 	siteClient, err := ns.nodeClient.Proxy.ConnectToSite()
 	if err != nil {
 		log.Error(err)
+		return
 	}
 	tick := time.NewTicker(defaults.SessionRefreshPeriod)
 	defer tick.Stop()

